Framework/Kernel/test/MersenneTwisterTest.h

```diff
@@ -139,10 +139,10 @@ class MersenneTwisterTest : public CxxTest::TestSuite {
 
 private:
   void assertSequenceCorrectForSeed_39857239(MersenneTwister &randGen) {
-    double expectedValues[10] = {
-        0.203374452656, 0.597970068222, 0.120683325687,  0.92372657801,
-        0.734524340136, 0.467380537419, 0.0712658402044, 0.204503614921,
-        0.487210249063, 0.885743656661};
+    double expectedValues[10] = {0.597970068269, 0.923726578038, 0.46738053759,
```

Comment:
A comment in https://github.com/TRIQS/triqs/issues/226#issuecomment-126280169 suggests that it is actually `uniform_real_distribution` that produces different results and the underlying engine produces the same numbers. This is probably something we will have to live with.